### ISSUE TITLE
How Activate a window

### ISSUE THREAD

**Issue by macdeport** [Kalmat/PyWinCtl#5]
I have tried without success:
```Python
import pywinctl
w=pywinctl.getWindowsWithTitle('/Users')[0] ;print(w)
w.activate()

```

**Comment by Kalmat** [comment]
Fixed!
Stupid mistake of mine between application and process. Besides, it always passes tests (it's already the active window!).

Thanks, mate!

**Comment by macdeport** [comment]
Thanks for you fix.

With:
```Python
import pywinctl
w=pywinctl.getWindowsWithTitle('/Users');print(w)
w=w[0]
print(w.isActive)
time.sleep(0.3)

w.activate()

```
The window effectively activate but I get these messages:
```
[MacOSWindow(hWnd=<NSRunningApplication: 0x7f9a7a50cbf0 (com.cocoatech.PathFinder - 13556)>)]
False
649:659: execution error: Can’t make missing value into type boolean. (-1700)
649:659: execution error: Can’t make missing value into type boolean. (-1700)

```

**Comment by Kalmat** [comment]
Hey! Thank you!

I'm affraid Path Finder has a limited Apple Script support and lacks of many usual attributes. If you test same code with, for example, Finder or Terminal, it works rigth with no complains.

In this case, the attribute PathFinder doesn't have is "visible", which is used by the show() method (which is invoked by activate(), and is also used by isVisible property). This is why the script complains. The fallback I use when an app is not able to use "visible" attribute for its windows is to hide/show the whole application... Not sure if it is the best behavior, by the way.

According to this, I had to make some adjustments in isActive property. Now it works with Path Finder (and hopefully with those other apps with limited AppScript capabilities too)! On the other hand, isVisible does not work and found no workaround for the inexistence of "visible" attribute (I also tried isHidden() method at application level, not window level, but it always returns "False"...). I will upload a new version (0.0.28) as soon as I can right today with this fix.

PS: Bear in mind you might need to set ''wait'' argument in order to give enough time for the WM to refresh the window state. This might be needed in all cases, but specially in Apple Script version since it is significantly slower.
